### ISSUE TITLE
Power hungry on OSX

### ISSUE THREAD

**Issue by lucalcin** [LaserWeb/LaserWeb4#468]
Hi,

LaserWeb4 in working fine on OSX, but I noticed that there is a very high power requirement even if LW4 is idle.
In my MacBook Pro LW4 uses 25%-80% of the computer resource when idle, and 100% when imports, export or calculate g-code.
I suppose there is some power-consuming routine that can be optimized
Please ask for further info/test
In attachment the task manager that shows LW4 in idle state consuming 63% of the resources.

![schermata 2018-04-23 alle 07 03 39](https://user-images.githubusercontent.com/28046872/39107793-d014be4e-46c4-11e8-96c6-40c77741c5cc.png)
![schermata 2018-04-23 alle 07 01 46](https://user-images.githubusercontent.com/28046872/39107794-d03c23ee-46c4-11e8-84fa-f8be07a2d4de.png)

**Comment by carlosen14** [comment]
same problem..

**Comment by lucalcin** [comment]
No news?
![schermata 2018-05-24 alle 21 47 35](https://user-images.githubusercontent.com/28046872/40510172-221857d8-5f9c-11e8-9d2b-d457471239bc.png)

**Comment by jorgerobles** [comment]
Cannot reproduce the issue:




2018-05-24 21:48 GMT+02:00 lucalcin <notifications@github.com>:

**Comment by jorgerobles** [comment]
Of course, LW is OpenGL intensive, I've got discrete video (Nvidia) that could be the fix for the issue.

**Comment by carlosen14** [comment]
@jorgerobles what graphic card do you have? I have this specs.. 
<img width="328" alt="captura de pantalla 2018-05-24 a la s 22 11 28" src="https://user-images.githubusercontent.com/9368632/40525780-71b3c41e-5f9f-11e8-9c61-5ebcc7223d49.png">

Maybe is the poor graphic card the one that complains and works harder.

**Comment by carlosen14** [comment]
@jorgerobles what graphic card do you have? I have this specs..
Maybe is the poor graphic card the one that complains and works harder.

<img width="309" alt="captura de pantalla 2018-05-24 a la s 22 12 59" src="https://user-images.githubusercontent.com/9368632/40525805-a56c466e-5f9f-11e8-9964-6b5bb7f5af21.png">

**Comment by jorgerobles** [comment]
It's on the pic :)

Nvidia gtx760.

**Comment by lucalcin** [comment]
![schermata 2018-05-25 alle 08 53 29](https://user-images.githubusercontent.com/28046872/40530446-1e45f66c-5ff9-11e8-8b84-cdd2ceae45bf.png)

**Comment by jorgerobles** [comment]
https://github.com/LaserWeb/LaserWeb4/issues/216

Enable settings/application/display cache
